realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -462,6 +477,15 @@ private static String getManagementRealmUrl(URL authUrl) {
         }
     }
 
+    /**
+     * FIXME Javadoc
+     *
+     * @return
+     */
+    public PermissionManager getPermissionManager() {
```

Comment:
I  could, but what would the point be? It wouldn't it reduce the swamp that is `io.realm`? Originally I had everything in `io.realm.permissions()`, but then you would be forced to have a public constructor which wasn't supposed to be used (gief Java 9 modules)